### PR TITLE
Create onboarding.md

### PR DESCRIPTION
**Pull request by supernord**
Suggesting content for an on boarding document that supports new users of WorkflowHub.
😃

### CODE REVIEW

File: docs/onboarding.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: How to get started with WorkflowHub
+---
+
+WorkflowHub will help make your workflows findable and citable, and this guide will help you get started with that process.
+
+## 1. [Register yourself on WorkflowHub](https://workflowhub.eu/signup)
+
+{% include callout.html type="note" content="you can also log in using your GitHub credentials." %}
```
Comment (ilveroluca) [inline docs/onboarding.md]: In addition to GitHub, we should also mention [LS Login](https://lifescience-ri.eu/ls-login/).

I think this should work:
```
{% include callout.html type="note" markdown="span"
content="You can also log in using your [LS Login](https://lifescience-ri.eu/ls-login/) or [GitHub](https://github.com/) credentials." %}

```

**Comment by bedroesb** [comment]
@fbacall @supernord Is this on boarding doc replacing the page on https://about.workflowhub.eu/docs/ ? or how will this fit in in the existing sidebar structure.

**Comment by fbacall** [comment]
We agreed in the club meeting that it would replace the "Before you start" root item thing, but I've not got around to doing it yet

**Comment by bedroesb** [comment]
So we get rid of the before-you-start/ page/url or we add the content of onboarding to that markdown? 

Sorry for the questions, couldn't make it to the meeting!

**Comment by fbacall** [comment]
We didn't discuss that much in detail, and I don't have an opinion either way.

Maybe we just redirect the "Before you start" link to go to `onboarding` (or will that break something?). Just thinking in case someone is already linking to `/docs/onboarding`...

**Comment by bedroesb** [comment]
Is there already linked towards the onboarding page? I think the opposite is more real but I don't mind that much. I would just either have in the sidemenu `Onboarding` as menu item, and than also as page title to have everything aligned, or everything `Before you start`. This  accompanied with a redirect from one of the others. Because right now both markdown files exit in the repo and that is not needed offcourse.

**Comment by supernord** [comment]
@fbacall @bedroesb - do you think `onboarding` is descriptive enough to be a menu item?

would any of the following work?

- `Getting started with WorkflowHub`
- `Getting started`
- `New users`
- `Orientation`

**Comment by bedroesb** [comment]
I still like getting started and would just add the content there :)

**Comment by fbacall** [comment]
I'm happy with whatever you guys decide

**Pull request merged**